tests/integration/kubernetes/k8s-initdata.bats

```diff
@@ -183,7 +183,7 @@ EOF
         return 1
     fi
 
-    assert_logs_contain "${node}" kata "${node_start_time}" "\[CDH\] \[ERROR\]: Get Resource failed"
+    assert_logs_contain "${node}" kata "${node_start_time}" "failed to pull image"
```

Comment:
We do not need to change this line.   see https://github.com/kata-containers/kata-containers/actions/runs/15007552372/job/42171904971?pr=11259#step:17:1871